src/Razor/Microsoft.NET.Sdk.Razor/src/build/netstandard2.0/LinkerWorkaround.xml

```diff
@@ -23,4 +23,7 @@
       <method signature="System.Void .ctor()" />
     </type>
   </assembly>
+
+  <!-- Workaround for https://github.com/mono/linker/issues/1298 -->
+  <assembly fullname="System.Security.Cryptography.X509Certificates" />
```

Comment:
FYI - @sbomer @vitek-karas. Blazor needs to preserve the assembly in order for the app to work.

This should be able to be removed once "member level" trimming is enabled, since the linker will remove the unused static field that requires X509Certificates.

Reply:
I'm fine with the workaround - we still need to figure out how come linker broke this...